generate_report.py

```diff
@@ -0,0 +1,218 @@
+import sys
+import os
+from collections import Counter
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tick
+import matplotlib.mlab as mlab
+import matplotlib.backends.backend_pdf
+
+regions = ["LON1", "AMS3", "FRA1", "NYC3", "BLR1", "SFO2", "NYC3-2", "SFO2-2", "SGP1", "TOR1", "AMS3-2", "FRA1", "LON1", "NYC3", "SFO2", "SGP1", "TOR1", "AMS3", "FRA1", "LON1", "NYC3", "SFO2", "SGP1", "TOR1"]
```

Comment:
Why do some regions repeat here and some don't (e.g. `TOR1` occurs 3 times while `AMS` twice and once as `AMS3-2`)?

Reply:
ahh :sweat_smile: The regions were copy pasted from the defaults in `./terraform/cluster/variables.tf`, and then indexes were added to make each droplet in the region unique. I was only using the first ten regions, so I only added indexes to the first 10 regions. I have added a function to make this more explicit, and perform it uniformly across all default regions.